sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/DigitalTwinsLifecycleSample.java

```diff
@@ -6,7 +6,9 @@
 import com.azure.core.http.policy.HttpLogDetailLevel;
```

Comment:
I think we should have an Async Suffix in the file name to indicate that we are using Async APIs for this sample.